doc/modules/cross_validation.rst

```diff
@@ -554,3 +554,20 @@ Cross validation and model selection
 Cross validation iterators can also be used to directly perform model
 selection using Grid Search for the optimal hyperparameters of the
 model. This is the topic of the next section: :ref:`grid_search`.
+
+Cross validation of time series data
+====================================
+
+Time series data is known for the correlation between observations, so 
+it is very important to evaluate our models on the "future" observations that 
+are not seen by our model before. However, the "classical" k-times 
+cross-validation technique is based on the fact that each observation in 
+the available data set is used (k-1)-times to train a model and 1 time to 
+test it. Therefore, we need to adopt the special time series cross-validation 
+to avoid the problem, which is described as follows:
+
+1. In each iteration :math:`i`, use first :mata:`i` fold(s) as training set, 
```

Comment:
I don't think steps help. Just do "We first split the data into `n_splits + 1` folds, then in each iteration ...". Drop points 2 (where we also need to fit, and consider other evaluation metrics) and 3.